container/rkt/handler.go

```diff
@@ -100,16 +100,16 @@ func newRktContainerHandler(name string, rktClient rktapi.PublicAPIClient, rktPa
 	if err != nil {
 		return nil, err
 	} else {
-		var annotations []*rktapi.KeyValue
-		if parsed.Container == "" {
-			pid = int(resp.Pod.Pid)
-			apiPod = resp.Pod
-			annotations = resp.Pod.Annotations
-		} else {
-			var ok bool
-			if annotations, ok = findAnnotations(resp.Pod.Apps, parsed.Container); !ok {
+		annotations := resp.Pod.Annotations
+		if parsed.Container != "" {
+			if contAnnotations, ok := findAnnotations(resp.Pod.Apps, parsed.Container); !ok {
 				glog.Warningf("couldn't find application in Pod matching %v", parsed.Container)
```

Comment:
nit: This error is a bit confusing. How about "couldn't find Apps annotation in Pod matching %v"